### PR TITLE
Working producer application that pulls in the consumers contracts

### PR DESCRIPTION
**Pull request by Budlee**
Working code for the

### CODE REVIEW

**Comment by Budlee** [comment]
Updated the PR

File: 02-contracts/Consumer-code/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# A Client that produces contracts
+
+This is a client application that is producing contracts.
+Here the consumer is creating a Pact that it is publishing to the Producer to fufil
```
Comment (jpgough) [inline 02-contracts/Consumer-code/README.md]: Publishes to the producer or for the Produer?

File: 02-contracts/Consumer-code/build.gradle [hunk 2]
```diff
@@ -0,0 +1,78 @@
+buildscript {
```
Comment (jpgough) [inline 02-contracts/Consumer-code/build.gradle]: Can we rename the folder this is all in from `Consumer-code` to `consumer-code`

File: 02-contracts/Consumer-code/src/test/java/com/budlee/contract/app/service/TestTodosServiceShould.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                .method("PUT")
+                .headers("Content-Type", "application/json")
+                .body("{\"message\":\"Make all the beds\"}")
+                .willRespondWith()
+                .status(204)
+                .toPact()
+                ;
+    }
+
+//    @Pact(consumer = "todo-api-consumer-pact-update-no")
```
Comment (jpgough) [inline 02-contracts/Consumer-code/src/test/java/com/budlee/contract/app/service/TestTodosServiceShould.java]: Remove commented out code?

File: 02-contracts/Consumer-code/target/pacts/todo-api-consumer-pact-todo-api-producer-pact.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
+{
```
Comment (jpgough) [inline 02-contracts/Consumer-code/target/pacts/todo-api-consumer-pact-todo-api-producer-pact.json]: /target - should this be checked in

File: 02-contracts/Producer-code/build.gradle [hunk 5]
```diff
@@ -0,0 +1,64 @@
+buildscript {
```
Comment (jpgough) [inline 02-contracts/Producer-code/build.gradle]: `Producer-code` can we rename to `producer-code`

File: 02-contracts/Consumer-code/src/test/java/com/budlee/contract/app/service/TestTodosServiceShould.java (hunk 3 above)
Comment (Budlee) [reply 02-contracts/Consumer-code/src/test/java/com/budlee/contract/app/service/TestTodosServiceShould.java]: Done

**Review by jpgough — APPROVED** [review]

**Pull request merged**